core/src/main/scala/pickling/Pickler.scala

```diff
@@ -84,6 +86,22 @@ trait Unpickler[T] {
 }
 object Unpickler {
   def generate[T]: Unpickler[T] = macro Compat.UnpicklerMacros_impl[T]
+  implicit def spuToU[T: SPicklerUnpickler]: Unpickler[T] = implicitly[SPicklerUnpickler[T]].unpickler
+}
+
+/** A combination of a static pickler and an unpickler for type `T`.
+ */
+trait SPicklerUnpickler[T] {
```

Comment:
I think I like the notion of this extending SPickler[T] and Unpickler[T] PRIMARILY for custom-written picklers.  a.k.a. it provides a convenience there that I think is worth it.